gui/gamesetup/gamesetup~!extra_commands.js

```diff
@@ -348,16 +349,58 @@ g_NetworkCommands['/countdown'] = input =>
 
 g_NetworkCommands['/gameName'] = text =>
 {
-	if (!g_IsController || !Engine.HasNetServer())
-		return;
-	if (!g_SetupWindow.controls.lobbyGameRegistrationController)
-		return
-	text = `${text}`
-	g_SetupWindow.controls.lobbyGameRegistrationController.serverName = text
-	selfMessage(`Game name changed to: ${text}`)
-	g_SetupWindow.controls.lobbyGameRegistrationController.sendImmediately()
+	return setGameNameInLobby(text);
+}
+
+
+g_NetworkCommands["/jitsi"] = text =>
+{
+	// selfMessage(`meet.jit.si/anyNameYoutWantHere`);
+	selfMessage(
+	  `Jitsi Meet is a fully encrypted, 100% open source video conferencing solution that you can use all day, every day, for free — with no account needed.`
+	)
+
+
+	let gitHubLinkAutoCivModificationSL5 = 'https://github.com/sl5net/autociv';
+
+	let textBest = `Jitsi is best for having a quick team call without setup process. could used as audio chat for you 0ad-team. `
+	selfMessage(
+		textBest
+	)
+
+	selfMessage(
+	  `write /link to open a link in your WebBrowser if you have autoCiv-mod installed to open a link more easy.`
+	)
+
+	let linkLong = `https://meet.jit.si/0ad${linkidShort}audio`
+
+	let linkTeam1example = `https://meet.jit.si/0ad${linkidShort}team123`
+	selfMessage(
+		` recommendation: send later in your private team-game-chat a other unique link for audio chat. Example:  ${linkTeam1example}`
+	)
+
+	selfMessage(
+		` this autoCiv-mod modification you could donwload here: ${gitHubLinkAutoCivModificationSL5}`
+	)
+
+
+	let population = 250
+	game.set.population(population)
+
+	g_GameSettings.startingResources.resources = 200
+	game.updateSettings()
+
+	selfMessage(linkLong)
+	openURL(linkLong) // openURL("https://webchat.quakenet.org/?channels=0ad")
+
+	// let gameTextJitsiExplainded = `want use Jitsi as a fully encrypted, open source video conferencing, with no account needed.`
+	// gameText = ` ${textBest} ${linkLong} `;
+	let gameText = `jitsi | ${textBest} ${linkLong}`
+	return setGameNameInLobby(gameText);
```

Comment:
This is also not part of the command

Reply:
the idea is that is visible from outside.